rosdep/base.yaml

```diff
@@ -3113,7 +3113,11 @@ libturbojpeg:
   fedora: [turbojpeg-devel]
   freebsd: [libjpeg-turbo]
   gentoo: [media-libs/libjpeg-turbo]
-  ubuntu: [libturbojpeg, libjpeg-turbo8-dev]
+  ubuntu: 
+    '*': [libturbojpeg, libjpeg-turbo8-dev]
```

Comment:
for future-proof-ness we may want to inverse the logic here to keep the wildcard for "any future distro". This will allow us to no have to add a new rule everytime a new distro comes out